styles/_variables.scss

```diff
@@ -69,6 +69,7 @@ $breakpoints: (
     'large' '(min-width: 1023px)',
     'menu-break' '(min-width: 1255px)',
     'hero-break' '(min-width: 1350px)',
+    'menu-break-larger' '(min-width: 1400px)',
     'site-width' '(min-width: 1441px)',
```

Comment:
This is one pixel off from `site-width`; would it be ok to use `site-width` instead of adding the new variable?

Reply:
good spot - will change on torchbox.com too